testdata/yaml/habitat-local-sd-command.yaml

```diff
@@ -0,0 +1,20 @@
+## Namespace for the command
+namespace: foo
+# Command name itself
+name: bar
+# Description of the command and what it does
+description: |
+  Lorem ipsum dolor sit amet.
+# Maintainer of the command
+maintainer: foo@bar.com
+# Major and Minor version number (patch is automatic)
+version: 1.0
+# Format the command is in (see below for examples)
+# Valid options: habitat, docker, binary
+format: habitat
+# Habitat specific config
+# if format: habitat
+habitat:
+    mode: local
+    package: ./testdata/binary/hello.hart
```

Comment:
We should need to add package information to execute the package (e.g. `python2/python`). `.hart` file does not have own package information.
So we also have to update command dataschema like below:

```
habitat:
    mode: local
    path: ./testdata/binary/hello.hart
    package: core/git
    command: git
```

Reference: https://github.com/screwdriver-cd/screwdriver/issues/677#issuecomment-383417331

Reply:
oh,, I had misread the comment. I will fix that way, thank you!!

Reply:
We should use `file` key instead of `path` like the binary format.